openwisp_notifications/handlers.py

```diff
@@ -62,14 +63,19 @@ def notify_handler(**kwargs):
     )
     verb = notification_template.get('verb', kwargs.pop('verb', None))
     target_org = getattr(target, 'organization_id', None)
+    user_model_name = swapper.split(
+        swapper.get_model_name('openwisp_users', 'Organization')
+    )[0]
```

Comment:
```suggestion
    user_app_name = User._meta.app_label
```

We are already loading `User` and `Organization` model using swapper. I don't think we need to do that again here.